apps/freeablo/faworld/world.cpp

```diff
@@ -185,6 +185,21 @@ namespace FAWorld
                                    "levels/towndata/town.min",
                                    "levels/towndata/town.sol",
                                    "levels/towndata/town.cel",
+                                   "levels/towndata/towns.cel",
+                                   std::map<int32_t, int32_t>{{357, 1},
+                                                              {128, 5},
```

Comment:
This probably shouldn't be hardcoded, but for now I think it's fine since we'll probably have to change how levels are stored on disk anyway at some point (I'd like to eventually get rid of the min/til/dun hierarchy and just have "tiles").
Maybe for now you could add a comment to this and the other palce with hardcoded mappings, explaining what they are used for?

Reply:
Also, actually, what does this do in the town level?

Reply:
Ok will add comments.
The town ones overlay on parts of trees, although they seems to make no difference to the final output...
Here's a screenshot with special cels highlighted:
![screen shot 2019-01-05 at 8 03 56 pm](https://user-images.githubusercontent.com/8636545/50730160-394df180-11ab-11e9-9438-e1ecab4f9f29.png)
And here's all the special cel images in case anyone is interested:
![levels-towndata-towns](https://user-images.githubusercontent.com/8636545/50730126-9b5a2700-11aa-11e9-8242-fb2751171881.png)
![levels-l1data-l1s](https://user-images.githubusercontent.com/8636545/50730124-9ac19080-11aa-11e9-8228-0549d152e515.png)
![levels-l2data-l2s](https://user-images.githubusercontent.com/8636545/50730125-9b5a2700-11aa-11e9-9683-95d6357afb48.png)